src/main/kotlin/no/nav/familie/ef/sak/vilkår/VurderingService.kt

```diff
@@ -179,6 +278,34 @@ class VurderingService(
         }
     }
 
+    private fun validerBehandlingForGjenbruk(
+        nåværendeBehandlingId: UUID,
+        tidligereBehandlingId: UUID,
```

Comment:
savner én sjekk: at behandlingen ikke er låst for videre redigering? (Eller er jeg blind?)

Reply:
Lagt til: [4dcf61e](https://github.com/navikt/familie-ef-sak/pull/1740/commits/4dcf61e78ec5e443e8ff36c6fe3e97ab37094ff1)